cmd/gossamer/flags.go

```diff
@@ -82,6 +82,11 @@ var (
 		Name:  "genesis",
 		Usage: "Path to genesis JSON file",
 	}
+	// GenesisHRFlag tells parser genesis.json contains human readable field, and to convert to raw
+	GenesisHRFlag = cli.BoolFlag{
+		Name:  "genesis_hr",
```

Comment:
maybe we should rename `--genesis` to `--genesis-raw` and this to `--genesis`, what do you think? I think that's clearer for people to use

Reply:
Regarding naming, I agree it's not that clear.  The `--genesis_hr` flag is a boolean that tells the parser the genesis file is human-readable.  But since `--genesis` is a flag for path it strange.  So, I agree I should change the current `--genesis` flag `--genesis-raw` and change `--genesis_hr` into `--genesis` and make it so it's a path to human-readable path.  Then change flags to check that only one is set.